examples/c/ex_backup_block.c

```diff
@@ -426,6 +457,9 @@ main(int argc, char *argv[])
 
     (void)snprintf(cmd_buf, sizeof(cmd_buf), "rm -rf %s && mkdir %s", home, home);
     error_check(system(cmd_buf));
+    (void)snprintf(
+      cmd_buf, sizeof(cmd_buf), "rm -rf %s/%s && mkdir %s/%s", home, logpath, home, logpath);
+    error_check(system(cmd_buf));
```

Comment:
I think this all can be a single `"rm -rf %s && mkdir -p %s/%s"`, instead of two `rm` and two `mkdir`.

Reply:
Fixed.